### PR TITLE
test background fetch

### PR DESCRIPTION
**Pull request by rasom**
[comment]: # (Please replace ... with your information. Remove < and >)
[comment]: # (To auto-close issue on merge, please insert the related issue number after # i.e fixes #566)

If you submit PR for issue with bounty then write here Fixes #NN where NN is issue number

*otherwise*

fixes #...

### Summary

[comment]: # (Summarise the problem and how the pull request solves it)
...

Documentation change PR (review please): https://github.com/status-im/status.im/pull/xxx

#### Platforms

- Android
- iOS
- macOS
- Linux
- Windows

##### Functional

- 1-1 chats
- public chats
- group chats
- wallet / transactions
- dapps / app browsing
- account recovery
- new account
- user profile updates
- networks
- mailservers
- fleet
- bootnodes

##### Non-functional

- battery performance
- CPU performance / speed of the app
- network consumption

### Steps to test

- Open Status
- ...
- Step 3, etc.

status: ready

### CODE REVIEW

**Comment by jakubgs** [comment]
I cleared the ios cert issue by removing the workspaces on all CI hosts. Now the error is:
```
/Users/jenkins/workspace/status-react_prs_ios_PR-12069/ios/StatusIm/AppDelegate.m:99:5: error: use of undeclared identifier 'TSBackgroundFetch' 
```

**Comment by rasom** [comment]
thanks @jakubgs

File: ios/Podfile [hunk 1]
```diff
@@ -34,12 +34,14 @@ abstract_target 'Status' do
...
-
   use_native_modules!
+
+  post_install do |installer_representation|
+    installer_representation.pods_project.targets.each do |target|
+        target.build_configurations.each do |config|
+            config.build_settings['ONLY_ACTIVE_ARCH'] = 'NO'
+        end
+    end
+  end
```
Comment (briansztamfater) [inline ios/Podfile]: Nice

**Comment by rasom** [comment]
I was almost ready to close the PR but it actually started working in PR build. Still can't simulate it because of this error https://github.com/transistorsoft/react-native-background-fetch/issues/336, but apparently the app is started in the background periodically

**Comment by flexsurfer** [comment]
hey @rasom still relevant ?

**Pull request closed without merging**